flocker/node/agents/blockdevice.py

```diff
@@ -1533,8 +1644,10 @@ class BlockDeviceDeployer(PClass):
     block_device_api = field(mandatory=True)
     _profiled_blockdevice_api = field(mandatory=True, initial=None)
     _async_block_device_api = field(mandatory=True, initial=None)
-    mountroot = field(type=FilePath, initial=FilePath(b"/flocker"))
-    shared_root = field(type=FilePath, initial=FilePath(b"/flocker"))
+    mountroot = field(type=FilePath,
+                      initial=FilePath(b"/var/flocker/mounts"))
+    link_root = field(type=FilePath, initial=FilePath(b"/flocker/v2"))
```

Comment:
Correct. I want there to be code that verifies that everything is set up correctly before I change these to be different values.

Even in this PR there are tests that set these to different values.